test/unit/object/test_events.py

```diff
@@ -205,6 +219,12 @@ def test_get_events(
         expected_url,
         params=dict(limit=100, stream_position=0, **expected_stream_type_params),
     )
+    event_entries = events['entries']
+    assert len(event_entries) == len(events_response.json.return_value['entries'])
+    if len(event_entries):
```

Comment:
this could be a walk over the events in the list of event entries.  That way if the events_response gets modified to have more that 1 event returned then this test still works.